RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -1,104 +1,79 @@
-     'keep recoSuperCluster_uncleanedHybridSuperClusters_*_*',
-     'keep recoSuperCluster_uncleanedOnlyCorrectedHybridSuperClusters_*_*',
-     'keep recoSuperCluster_uncleanedOnlyCorrectedMulti5x5SuperClustersWithPreshower_*_*',
-     'keep recoSuperCluster_uncleanedOnlyMulti5x5SuperClustersWithPreshower_*_*',
-    #'keep recoCaloClusters_*_*_*',
-     'keep recoCaloClusters_particleFlowEGamma_*_*',
-     'keep recoCaloClusters_cleanedHybridSuperClusters_*_*',
-     'keep recoCaloClusters_hybridSuperClusters_*_*',
-     'keep recoCaloClusters_uncleanedHybridSuperClusters_*_*',
-     'keep recoCaloClusters_islandBasicClusters_*_*',
-     'keep recoCaloClusters_multi5x5BasicClustersCleaned_*_*',
-     'keep recoCaloClusters_multi5x5BasicClustersUncleaned_*_*',
-     'keep recoCaloClusters_multi5x5SuperClusters_*_*',
-     'keep recoCaloClusters_particleFlowSuperClusterECAL_*_*',
-     'keep recoCaloClusters_multi5x5SuperClusters_*_*',
-    #'keep EcalRecHitsSorted_*_*_*',
-    'keep EcalRecHitsSorted_ecalRecHit_*_*',
-    'keep EcalRecHitsSorted_ecalPreshowerRecHit_*_*',
-    'keep EBSrFlagsSorted_ecalDigis__*',
-    'keep EESrFlagsSorted_ecalDigis__*',
-    #'keep floatedmValueMap_*_*_*',  # isolation not created yet in RECO step, but in case it is later
-    'keep floatedmValueMap_hiDetachedTripletStepQual_MVAVals_*',        
-    'keep floatedmValueMap_hiDetachedTripletStepSelector_MVAVals_*',    
-    'keep floatedmValueMap_hiGeneralTracks_MVAVals_*',                  
-    'keep floatedmValueMap_hiInitialStepSelector_MVAVals_*',            
-    'keep floatedmValueMap_hiLowPtTripletStepQual_MVAVals_*',           
-    'keep floatedmValueMap_hiLowPtTripletStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiPixelPairStepSelector_MVAVals_*',          
-    'keep floatedmValueMap_hiRegitMuInitialStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiRegitMuMixedTripletStepSelector_MVAVals_*',
-    'keep floatedmValueMap_hiRegitMuPixelLessStepSelector_MVAVals_*', 
-    'keep floatedmValueMap_hiRegitMuPixelPairStepSelector_MVAVals_*', 
-    #'keep recoPFCandidates_*_*_*',
-    'keep recoPFCandidates_particleFlowEGamma_*_*',
-    'keep recoPFCandidates_particleFlowTmp_*_*',
-    "drop recoPFClusters_*_*_*",
-    #"keep recoElectronSeeds_*_*_*",
-    "keep recoElectronSeeds_ecalDrivenElectronSeeds_*_*", 
-    #"keep recoGsfElectrons_*_*_*",
-    "keep recoGsfElectrons_ecalDrivenGsfElectrons_*_*",
-    "keep recoGsfElectrons_electronsWithPresel_*_*",
-    "keep recoGsfElectrons_gedGsfElectronsTmp_*_*",
-    "keep recoGsfElectrons_mvaElectrons_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
-    'keep recoPhotons_gedPhotonsTmp_*_*',
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerGED_*_*',
-    'keep recoElectronSeeds_ecalDrivenElectronSeeds_*_*',
-     'keep recoTrackExtras_electronGsfTracks_*_*'
-    )
-    )
+        'keep recoSuperClusters_correctedHybridSuperClusters_*_*',        
```

Comment:
You are removing several recoSuperCluster's wrt what was in the original code: @mandrenguyen could you please check and confirm that they are not needed? In doubt, I would put them back, since this PR is expected to be mostly technical

Reply:
Also for the recoSuperClusters and recoCaloClusters, 
based on the 140.56 step2(AOD) event content list, 
I've removed some keep commands that did not exist in the step2 AOD event contents :
```
    'keep recoSuperCluster_cleanedHybridSuperClusters_*_*',
    'keep recoSuperCluster_correctedEndcapSuperClustersWithPreshower_*_*',  
    'keep recoSuperCluster_islandSuperClusters_*_*',	
    'keep recoSuperCluster_mergedSuperClusters_*_*',	
    'keep recoSuperCluster_multi5x5SuperClustersCleaned_*_*',	        
    'keep recoSuperCluster_multi5x5SuperClustersUncleaned_*_*',
    'keep recoSuperCluster_uncleanedHybridSuperClusters_*_*',	
    'keep recoSuperCluster_uncleanedOnlyCorrectedHybridSuperClusters_*_*',	
    'keep recoSuperCluster_uncleanedOnlyCorrectedMulti5x5SuperClustersWithPreshower_*_*',	
    'keep recoSuperCluster_uncleanedOnlyMulti5x5SuperClustersWithPreshower_*_*',
```
and added some keep commands that exist in the step2 AOD event contents :
```
        'keep recoSuperClusters_hfEMClusters_*_*',
        'keep recoSuperClusters_particleFlowSuperClusterOOTECAL_*_*',
        'keep recoCaloClusters_particleFlowSuperClusterOOTECAL_*_*',
        'keep recoCaloClusters_lowPtGsfElectronSuperClusters_*_*',
        'keep recoCaloClusters_pfElectronTranslator_*_*',
        'keep recoCaloClusters_pfPhotonTranslator_*_*',
        'keep recoCaloClusters_hfEMClusters_*_*',
```
In RECO, I also fixed some typoes in type. 
they missing **"s"**  ('keep recoSuperCluster ->  'keep recoSuperCluster**s**).